frontend/pages/views/PhotoView.js

```diff
@@ -461,9 +461,11 @@ export class PhotoView extends PhotoViewer {
                         </div>
 
                         {tagList.map((word) => (
-                            <button className="tag-button" key={word.id}>
-                                {word}
-                            </button>
+                            <a key={"tag"} href={`/tag/${word.toLowerCase()}/`}>
```

Comment:
no need to lowercase if we're later title casing these, let's just use CSS to do this instead